src/integration-tests/introspector/mii/models/domain1.yaml

```diff
@@ -35,6 +35,37 @@ topology:
                     PublicAddress: '@@PROP:T3CHANNEL2_PUBLIC_LISTEN_ADDR@@'
                 T3Channel3:
                     ListenPort: 30014
+                'istio-probe':
```

Comment:
This looks odd... The customer shouldn't have to add these channels to the model.  What is this testing?

Reply:
This is just to test the 'replace' option if the user just happened already have this defined in their model (e.g. previous verazzano wdt models that have public address set to something else)

Reply:
This is no longer a "replace option" test, correct? ... since the name doesn't match. I think this is just confusing :)

Reply:
Correct, forgot I added this, removed